src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const resPromise = new Promise((resolve) => {
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+const rejPromise = new Promise((resolve, reject) => {
+  window.setTimeout(reject, 3000);
```

Comment:
The `reject` function should be called with an error message or reason for rejection. Consider passing a message to `reject`, like `reject('Timeout occurred')`, to provide more context in the rejection handler.